### DISCUSSION TITLE
Всплывающее окно

### DISCUSSION THREAD

**Discussion by Spyker086** [totumonline/totum-ru-issues-and-discussions#520] [Как это сделать?] [answered]
Добрый день!
Может подскажете, можно ли организовать всплывающее окно после авторизации в системе с поздравлением 8 марта ?
Ну и окно это должно всплывать не для всех юзеров.

**Comment by erpic-ru** [comment, marked as answer]
Если у вас нет поле с полом сотрудника и много пользователей, то можете через чатгпт попределить его в тотуме по ФИО :), а затем просто отправить нотификацию всем сотрудницам, благо html там поддерживается

**Reply by Spyker086** [reply]
Ок, спасибо!
